omero/sysadmins/server-backup-and-restore.txt

```diff
@@ -79,7 +78,7 @@ OMERO.server has three main backup sources:
 You should back up *(1)* and *(2)* regularly. 
 
 .. warning:: **Although losing your PostgreSQL database is not as 
```

Comment:
Maybe move the "You should back up 1 and 2 regularly" into the warning, which could be "You must back up (1) and (2) regularly".  That's really all the information we need to convey, and it would make the point a bit more strongly.